docs/sphinx/source/installation.rst

```diff
@@ -38,23 +56,30 @@ For best compatibility, deploy in an `Anaconda 2.7` environment, or run::
 
 STEP 2
 ~~~~~~
-Move gencumulativesky.exe
+Windows:
 
 * Copy gencumulativesky.exe from the repo's ``/bifacial_radiance/data/`` directory and copy into your Radiance install directory.
   This is typically found in ``/program files/radiance/bin/``.  
  
+Linux/Mac OSX:
+
+* Copy the gencumulativesky executable from the repo's ``/bifacial_radiance/data/`` directory and copy into your Radiance install directory.
+  This is typically found in ``/usr/local/radiance/bin/``. 
+* For Linux/Mac OSX, you will need to install QT for the GUI to work properly. Installation and details described in (:issue:`130`:).
+
+
 .. note::
         GenCumulativeSky is detailed in the publication "Robinson, D., Stone, A., Irradiation modeling made simple: the cumulative sky approach and its applications, Proc. PLEA 2004, Eindhoven 2004."   
 
-The source is `available from the authors here <https://documents.epfl.ch/groups/u/ur/urbansimulation/www/GenCumSky/GenCumSky.zip>`_
- 
+        The gencumsky source is included in the repo's ``/bifacial_radiance/data/gencumsky`` directory along with a make_gencumskyexe.py script which builds the multi-platform gencumulativesky executables. More details on the use of this script in readme.txt or on thread (:issue:`1821`).
```

Comment:
As with the QT issue, installation instructions should be complete on their own. Pointing to another file not part of the documentation or to an issue, which is by definition something that should be solved, eventually, isn't that great. Besides, it is issue #182. #1821 does not exist.

Reply:
There's no need to put the instructions here, if you don't want to, but the PR number should be corrected.

Reply:
fixed!